client/src/components/common/names-capture.vue

```diff
@@ -803,7 +809,8 @@ export default class NamesCapture extends Vue {
 }
 
 .launch-icon {
-  font-size: 1rem;
+  display: inline-block;
+  font-size: 0.875rem;
```

Comment:
Second, change its display property.

Third, make it the same size as the text around it.